com.unity.render-pipelines.high-definition/Runtime/Material/Decal/DecalUtilities.hlsl

```diff
@@ -261,3 +261,25 @@ DecalSurfaceData GetDecalSurfaceData(PositionInputs posInput, float3 vtxNormal,
 
     return decalSurfaceData;
 }
+
+DecalSurfaceData GetDecalSurfaceData(PositionInputs posInput, FragInputs input, inout float alpha)
+{
+    float3 vtxNormal = input.tangentToWorld[2];
+    DecalSurfaceData decalSurfaceData = GetDecalSurfaceData(posInput, vtxNormal, alpha);
+
+#ifdef _DOUBLESIDED_ON
+    if (!input.isFrontFace)
+    {
+        if (_DoubleSidedConstants.x == -1.0f)
+            decalSurfaceData.normalWS.xyz *= -1.0f;
```

Comment:
this case is for flip mode, but I mirror the normal. In the case below for mirror mode, I flip the normal in tangent space (which seems quite expensive btw)
I don't know why it works but comparing the result with a double sided material, this is the only way to make them equivalent